libs/common/src/main/java/org/opensearch/common/lease/Releasable.java

```diff
@@ -30,7 +30,7 @@
  * GitHub history for details.
  */
 
-package org.opensearch.core.common.lease;
+package org.opensearch.common.lease;
 
 import java.io.Closeable;
 
```

Comment:
Line 38 should change to "Specialization of {@link AutoCloseable} that may not throw a checked exception" since BaseOpenSearchException is no longer available to this class. (the comment wasn't correct anyway)